docs/spectrum_collection.rst

```diff
@@ -0,0 +1,102 @@
+===================
+Spectrum Collection
+===================
+
+A spectrum collection is a way to keep a set of :class:`~specutils.Spectrum1D`
+objects together, regardless of their individual dispersion solutions. This is
+done by resampling the spectra onto a defined output grid.
+
+.. code:: python
+
+    >>> from specutils import Spectrum1D, SpectrumCollection
+    >>> import astropy.units as u
+    >>> import numpy as np
+    >>> spec1 = Spectrum1D(spectral_axis=np.linspace(0, 50, 50) * u.AA, flux=np.random.randn(50) * u.Jy)
+    >>> spec2 = Spectrum1D(spectral_axis=np.linspace(0, 50, 25) * u.AA, flux=np.random.randn(25) * u.Jy)
+    >>> spec_coll = SpectrumCollection([spec1, spec2])
+    >>> print(spec_coll.output_grid)
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    from specutils import Spectrum1D, SpectrumCollection
+    import astropy.units as u
+
+    spec1 = Spectrum1D(spectral_axis=np.linspace(0, 50, 50) * u.AA,
+                       flux=np.random.randn(50) * u.Jy)
+    spec2 = Spectrum1D(spectral_axis=np.linspace(0, 50, 25) * u.AA,
+                       flux=np.random.randn(25) * u.Jy)
+    spec_coll = SpectrumCollection([spec1, spec2])
+
+    f, (ax1, ax2) = plt.subplots(2, 1)
+
+    ax1.plot(spec1.wavelength, spec1.flux, label="High-res")
+    ax1.plot(spec2.wavelength, spec2.flux, label="Low-res")
+    ax2.plot(spec_coll.wavelength[0], spec_coll.flux[0])
+    ax2.plot(spec_coll.wavelength[1], spec_coll.flux[1])
+
+:class:`~specutils.SpectrumCollection` objects can be treated just like
+:class:`~specutils.Spectrum1D` objects; calling a particular attribute on the
+object will return an array whose type depends on the type of the attribute in
+the :class:`~specutils.Spectrum1D` object.
+
+.. code:: python
+
+    >>> print(type(spec1.flux))
+    astropy.units.quantity.Quantity
+    >>> print(type(spec_coll.flux))
+    astropy.units.quantity.Quantity
+
+The difference is their shape. The returned array from the :class:`~specutils.SpectrumCollection`
+object will have shape ``(N, M)`` where ``N`` is the number of input spectra
+and ``M`` is the length of the output dispersion grid.
+
+.. code:: python
+
+    >>> print(spec1.flux.shape)
+    (25,)
+    >>> print(spec_coll.flux.shape)
+    (2, 24)
+
+The items stored in the :class:`~specutils.SpectrumCollection` object are the
+*original* input spectra. Iterating over a :class:`~specutils.SpectrumCollection`
```

Comment:
capslock

Reply:
I'm not sure I understand this comment? 😅

Reply:
originaL -> original

Reply:
I think I might be blind... ? It all looks lowercase to me.

Reply:
this... might be something weird about github then?

![image](https://user-images.githubusercontent.com/143715/43266538-e3c5626c-90a8-11e8-83d4-cc605f650a27.png)


you see that L at the end?  But I think you're right, it's all correct in the text...

Reply:
Super weird. Might be a type hinting artifact?